src/Common/src/Interop/Linux/cgroups/Interop.cgroups.cs

```diff
@@ -125,7 +166,7 @@ private static bool TryFindHierarchyMount(string subsystem, out string root, out
                         while ((line = reader.ReadLine()) != null)
                         {
                             // Look for an entry that has cgroup as the "filesystem type"
-                            // and that has options containing the specified subsystem.
+                            // and, for cgroup1, that has options containing the specified subsystem
```

Comment:
For cgroup v2, no subsystem is included in the mount info; there's only a single global mount point. For example:

```
$ cat /proc/self/mountinfo | grep cgroup2
27 20 0:24 / /sys/fs/cgroup rw,nosuid,nodev,noexec,relatime shared:4 - cgroup2 cgroup2 rw,seclabel,nsdelegate
```